### PR TITLE
Update the quote difference interaction with style change on option

### PR DESCRIPTION
**Pull request by ronaldsg20**
**Before**
image

**After**
image

image

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/pegout/components/QuoteDiffDialog.vue [hunk 1]
```diff
@@ -2,45 +2,25 @@
...
+              <p class="justify-center">
+              Please note that the values are quotations and may undergo market changes.
+              </p>
+            </v-row>
           </v-col>
           <v-row class="mx-0 mb-8 mt-3" justify="space-around">
             <v-col class="d-flex justify-center">
               <v-btn-rsk @click="send">
-                <span>Continue</span>
+                <span>ok</span>
```
Comment (lserra-iov) [inline src/pegout/components/QuoteDiffDialog.vue:23]: why lowercase? 😭

**Review by lserra-iov — APPROVED** [review]

**Review by annipi — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**